packages/host/app/services/billing-service.ts

```diff
@@ -51,7 +52,13 @@ export default class BillingService extends Service {
   }
 
   get customerPortalLink() {
-    return this.stripeLinks.value?.customerPortalLink;
+    if (!this.subscriptionData?.stripeCustomerEmail) {
+      return this.stripeLinks.value?.customerPortalLink?.url;
+    }
+    const encodedEmail = encodeURIComponent(
+      this.subscriptionData.stripeCustomerEmail,
+    );
+    return `${this.stripeLinks.value?.customerPortalLink?.url}?prefilled_email=${encodedEmail}`;
```

Comment:
What happens for existing users who do not have their email saved? Will the link be `...link...?prefilled_email=undefined`? If yes that needs fixing